src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h2 {
```

Comment:
better to create class and style it

Reply:
Create class for body, h1 or h2? Or for all of that?

Reply:
Fixed